app/src/main/java/com/example/stromtracker/ui/raeume/RaeumeFragment.kt

```diff
@@ -41,10 +37,7 @@ class RaeumeFragment: Fragment() {
             }
         )
     }
-    private fun initRaum () {
-        var raum: Raum = Raum("Wohnzimmer",1)
```

Comment:
Ist auch schonj in develop drin, musst du bei dem Mergen dann aufpassen

Reply:
Du meinst wurde da schon gelöscht oder?

Reply:
ja genau